docs/list.md

```diff
@@ -333,6 +333,16 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### N
 
+#### **Native Script**
+
+- **Swag**: Free Swag
+- **Requirements**:
+    - Create a neat [StackBlitz](preview.nativescript.org) or Contribute a quality PR.
```

Comment:
```suggestion
    - Create a neat [StackBlitz](https://preview.nativescript.org) or Contribute a quality PR.
```
Hi @itsBaivab! Just letting you know, you need to have HTTP or HTTPS in the link parenthesis so that the markdown -> HTML conversion works. Otherwise, it think you are referencing an internal link, and will actually create a link to `https://deploy-preview-767--hacktoberfest-swag-list.netlify.app/list/preview.nativescript.org`. 

I'll fix this for you now, just wanted to make you aware to use full URLs going forward. 👍